Update group-analytics.mdx (#2817)
PostHog/posthog.com: contents/docs/user-guides/group-analytics.mdx

    </TabPane>
</Tabs>

## Renaming group types

> You can change how group types are displayed in insights and elsewhere in PostHog under project settings.

Reviewer: ```suggestion
You can change how group types are displayed in your Insights interface and throughout PostHog on Project settings.
```
